### DISCUSSION TITLE
Wall Outlet Output Turns off (State remains On).

### DISCUSSION THREAD

**Discussion by gid204** [arendst/Tasmota#15708] [Q&A] [answered]
I have recently purchased and flashed the following device with Tasmota v11.1.0
https://templates.blakadder.com/brilliant_BL20925.html
https://www.officeworks.com.au/shop/officeworks/p/brilliant-lighting-smart-wifi-plug-with-energy-monitoring-bl20925

I apply the template provided in the link above.
I can toggle the relay (I can hear it click) on and off via the button as well as via the web gui.

My issue is, whenever I plug any device in (large load or small) it powers up for about 2 seconds and then powers off. The state of the plug is still 'on' from Tasmota's perspective except current isn't being provided to the load. I.e. Tasmota UI says on, device LED indicator is still lit, indicating on but the load isn't powered.

I have a hunch from some research based on similar devices that there may be two relays, for active and neutral. Which means perhaps the template is wrong - I could be completely wrong here though.

Any help and guidance is really appreciated.

Cheers!

**Comment by gid204** [comment]
I have followed the first step from the following link and found that it seems I was wrong and there is only one relay.
https://tasmota.github.io/docs/Configuration-Procedure-for-New-Devices/

So the template does appear to be right....
So back to the original question, why would the output be shutting off immediately after I turn it on?

Thanks!

**Reply by sfromis** [reply]
Images here looks a lot like two relays, but no details of pins. A plausible setup would be that both relays are controlled by the same gpio, but I have zero knowledge of the design.
https://designtech.blogs.auckland.ac.nz/brilliant-smart-wifi-plug/

You should not use the example calibration commands there, the calibration should be individual for each plug, to deal with variance in components.

Do check status in the Tasmota console, web UI updating may be delayed. Better refresh the page for current data, both places.

**Comment by barbudor** [comment]
I it plausible indeed that the 2 relay are used to cut both main lines which is safe when you don't know which one is LINE and which one is NEUTRAL.

To understand why the relay turns off after 2 seconds, definitively console logs with `WebLog 3` would probably provides clues on the source of the switch-off

**Reply by gid204** [reply]
Unfortunately console logs isn't showing anything. The state of the plug is staying on its just the power out is being cut off. To clarify, the relay isn't 'clicking' off, when the power drops out. The power just stops. 

1. I turn output on (via button or web gui). Output turns on, console shows power on.
2. Power stops coming out of the plug however web gui says on, LED on device is still lit (on) and web console does not report it being turned off (i.e. its still on).
3. If I power off I then hear the relay click, web gui shows off, web console reports off.

I've downgraded down to 9.1 then to 8.5.1 to see if that would make any difference, it did not.

I'm starting to suspect a faulty relay or other component.

**Comment by joba-1** [comment]
do you have another power measurement device? I think there is a short inside and a overheat protection (like thermo resistor) kicks in

**Reply by gid204** [reply]
I don't unfortunately...

Although the engineer in me wants to work it out, I think it's worth my time just to take it back for a replacement/buy a new one given how cheap they are.

Happy to listen to all other suggestions but it's definitely sounding like a hardware fault....

**Reply by joba-1** [reply]
sounds reasonable. 
And maybe your replacement device can measure power :)

**Reply by rapi3** [reply]
I bought few online ( without warranty ) which had a defective thermal fuse, relay click but no power at all... I replaced it with a shunt and it works ok.

**Reply by joba-1** [reply]
well, if you can send it back it’s the best option. 
I would not remove an important safety feature! If anything, I‘d replace the faulty thermal fuse with a working one.

**Reply by rapi3** [reply]
I use it to power on/off a low load IT equipment ( max 50w ) so it is no problem ( relay inside it is rated for 16Amp / 3000w ) and I can't find a thermal fuse to fit inside for small EU version - Avatar AWP08L.

**Reply by joba-1** [reply]
Should work. Until something unexpected happens…

**Comment by gid204** [comment, marked as answer]
Thanks for all your comments, closing this off.
Solution = replacement.
